scss/components/_dropdowns.scss

```diff
@@ -99,7 +118,10 @@
 
       &.is-checked {
         @include u-icon-bg($check, $green-light);
-        background-position: 98%;
+        background-color: $gray-light;
+        background-position: 96%;
```

Comment:
Let's use the new CSS shorthand I just learned! `background-position: right 1rem;` or whatever the offset from the right side is.